src/board.cpp

```diff
@@ -0,0 +1,223 @@
+#include "board.h"
+#include "block.h"
+
+constexpr int movesROW[4] = {0, +1, 0, -1};
+constexpr int movesCOL[4] = {-1, 0, +1, 0};
```

Comment:
Weź zrób osobny struct (ze swoimi plikami) o nazwie Position, żeby miał po prostu pola x i y, zamiast za każdym razem pisać takie dziwne rzeczy

Reply:
Ten struct poza tym się jeszcze przyda

Reply:
Jak ma wyglądać ten struct? Nie wiem co masz dokładnie na myśli.